test/config_parser_SUITE.erl

```diff
@@ -12,6 +12,9 @@
 
 -define(HOST, <<"myhost">>).
 
+-define(eqf(Expected, Actual), ?assertEqual(Expected, apply(hd(Actual), [?HOST]))).
+-define(errf(Expr), ?assertError(_, apply(hd(Expr), [?HOST]))).
```

Comment:
These are very tricky, especially if `Actual` contains more elements, all but the head would get unnoticed... maybe you could use `eq_host_config` and `err_host_config` which also check the per-host case? See the latest `toml-config` revision.